netutils/ip.py

```diff
@@ -387,3 +389,64 @@ def get_usable_range(ip_network):
         lower_bound = str(net[1])
         upper_bound = str(net[-2])
     return f"{lower_bound} - {upper_bound}"
+
+
+# Python 3.6 lacks of ipaddress.IPv[46]Network.subnet_of and supernet_of methods:
+IPNetwork = Union[ipaddress.IPv4Network, ipaddress.IPv6Network]
+
+
+def ipaddress_subnet_of(this: IPNetwork, other: IPNetwork) -> bool:
```

Comment:
In the spirit of the project, we have stayed away from non-json serializable objects, meaning we should expect a string, and than check and convert to such an object.

Reply:
Can we change the name to something that will resonate better? 

Subnet/supernet?

Reply:
To be honest, names are coming from the ``ipaddress`` module itself. These functions are here just to support Python3.6. These methods exist in newer Python.